app/controllers/admin/users_controller.rb

```diff
@@ -37,7 +51,16 @@ def update
     else
       redirect_to admin_users_path
     end
+  end
+
+  def edit
+    @user = User.find(params[:format])
```

Comment:
the `params[:format]` suggests to me that something is wonky here. I think this is related to using the `:url` hash in the edit form below. Are you able to use `params[:id]` or similar?